src/Components/HeaderSolver.tsx

```diff
@@ -0,0 +1,28 @@
+import AppBar from "@mui/material/AppBar";
+import Button from "@mui/material/Button";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+
+type HeaderProps = {
+  title: string
+}
+
+export const Header = ({ title }: HeaderProps) => {
```

Comment:
What is the difference between using this and simply using the Header.tsx? They look very similar

Reply:
Gives the option to make the headers different. We talked about highlighting the open page in the header.

Reply:
The HeaderSolver.tsx has been deleted.